tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Scripts.spec.ts

```diff
@@ -13,65 +13,96 @@ test.describe('Script tests', () => {
 
   test('can create a empty script', async ({umbracoApi, umbracoUi}) => {
     // Arrange
-    await umbracoApi.script.ensureNameNotExists(scriptPath);
+    await umbracoApi.script.ensureNameNotExists(scriptName);
 
     // Act
     await umbracoUi.script.clickActionsMenuAtRoot();
-    await umbracoUi.script.clickNewScriptButton();
+    await umbracoUi.script.clickCreateThreeDotsButton();
+    await umbracoUi.script.clickNewJavascriptFileButton();
+    // TODO: Remove this timeout when frontend validation is implemented
+    await umbracoUi.waitForTimeout(500);
+    await umbracoUi.script.enterScriptName(scriptName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await umbracoUi.waitForTimeout(500);
+    await umbracoUi.script.clickSaveButton();
+
+    // Assert
+    await umbracoUi.script.isSuccessNotificationVisible();
+    expect(await umbracoApi.script.doesNameExist(scriptName)).toBeTruthy();
+
+    // Clean
+    await umbracoApi.script.ensureNameNotExists(scriptName);
+  });
+
+  test('can create a script with content', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.script.ensureNameNotExists(scriptName);
+    const scriptContent = 'TestContent';
+
+    // Act
+    await umbracoUi.script.clickActionsMenuAtRoot();
+    await umbracoUi.script.clickCreateThreeDotsButton();
+    await umbracoUi.script.clickNewJavascriptFileButton();
+    // TODO: Remove this timeout when frontend validation is implemented
+    await umbracoUi.waitForTimeout(500);
     await umbracoUi.script.enterScriptName(scriptName);
+    await umbracoUi.script.enterScriptContent(scriptContent);
     // TODO: Remove this timeout when frontend validation is implemented
-    await umbracoUi.waitForTimeout(1000);
+    await umbracoUi.waitForTimeout(500);
     await umbracoUi.script.clickSaveButton();
 
     // Assert
-    // TODO: Uncomment when the notification is visible
-    // await umbracoUi.isSuccessNotificationVisible();
-    expect(await umbracoApi.script.doesNameExist(scriptPath)).toBeTruthy();
+    await umbracoUi.script.isSuccessNotificationVisible();
+    expect(await umbracoApi.script.doesNameExist(scriptName)).toBeTruthy();
+    const scriptData = await umbracoApi.script.getByName(scriptName);
+    expect(scriptData.content).toBe(scriptContent);
 
     // Clean
-    await umbracoApi.script.ensureNameNotExists(scriptPath);
+    await umbracoApi.script.ensureNameNotExists(scriptName);
```

Comment:
To ensure that this step is executed regardless of whether the test fails, it is recommended to include it within the "afterEach" method.